app/controllers/prisoners_controller.rb

```diff
@@ -38,6 +44,12 @@ def image
 
 private
 
+  def fetch_pom_name(staff_id)
+    pom_firstname, pom_secondname =
+      PrisonOffenderManagerService.get_pom_name(staff_id)
+    "#{pom_secondname.titleize}, #{pom_firstname.titleize}"
```

Comment:
This formatting rule is in multiple places - is there a good way to DRY it up? (might be beyond the scope of this PR)

Reply:
Moved this method to application_helper.rb so it can be used across controllers, not 100% sure this was what you were thinking here?

Reply:
That makes it live in a single place, which is good!

My worry with putting it in `application_helper` is that it's quite general - if we do this with lots of things we end up with all sorts of functionality mixed together. That can make it hard to read/understand in the future.

I like to associate things like this with a particular model, so I have some `PrisonerOffenderManager` object, and I can ask it for its `.name`, making it the responsibility of that class to do any formatting/retrieving/orchestrating*. Because of the structure of the code right now, this would probably take too much work for this PR, though?

*(if one of these tasks became a large enough job, I'd happy split it further into presenters/repositories/whatever)

Reply:
Moved the method to the pom_helper, so it's still available centrally, but in a helper specifically related to POMs